plugin.xml

```diff
@@ -40,60 +40,39 @@
         <config-file target="res/values/strings.xml" parent="/resources">
             <string name="google_app_id">$SENDER_ID</string>
```

Comment:
@jcesarmobile do we need this anymore? Is it just contained in the google-services.json file?